src/elements/form-field/form-field/form-field.visual.spec.ts

```diff
@@ -0,0 +1,303 @@
+import { html, nothing, type TemplateResult } from 'lit';
```

Comment:
I'm not sure whether the focus is always where it's meant to be. Do you really want to test the mini button focus or was the goal to focus the input. In the second case you could use the method "withStateElement()" and pass the input element. there would be an example in the typography.visual.spec.ts file.

Reply:
Removed the focus from the states array and moved the tests with button/popover apart